link/uprobe.go

```diff
@@ -316,8 +312,29 @@ func tracefsUprobe(args probeArgs) (*perfEvent, error) {
 }
 
 // sanitizedSymbol replaces every invalid character for the tracefs api with an underscore.
-func sanitizedSymbol(symbol string) string {
-	return rgxEventSymbol.ReplaceAllString(symbol, "_")
+// It is equivalent to calling regexp.MustCompile("[^a-zA-Z0-9]+").ReplaceAllString("_").
+func sanitizedSymbol(s string) string {
+	var b strings.Builder
```

Comment:
Can you use https://pkg.go.dev/strings#Map instead?

Reply:
This will slightly change the output -- currently, each sequence of symbols that are not allowed are replaced with a single `_`.

If this is an OK change to made, let me know and I will do it.

Reply:
I think that is acceptable.